gatling-core/src/main/scala/io/gatling/core/json/Json.scala

```diff
@@ -45,8 +49,8 @@ object Json {
   }
 
   private def writeString(s: String, rootLevel: Boolean) = {
-    val escapedLineFeeds = s.replace("\n", "\\n")
-    if (rootLevel) fast"$escapedLineFeeds" else fast""""$escapedLineFeeds""""
+    val escapedLineFeeds = objectMapper.writeValueAsString(s).replaceAll("^\"|\"$", "")
```

Comment:
Jackson (correctly) adds double quotes to string as it is stringified.
`This is my string` will result in `"This is my string"` 

The purpose is of course that it aims to achieve the final goal being:
`key: "This is my string"` 

However in gatling double quotes are added later (ie writeMap) irrespective of the type. And so we would end up with `key: ""This is my string""` which is wrong. The reqex removes the double quoted at both ends.